### PR TITLE
Supplies back to front

### PR DESCRIPTION
**Pull request by Quentin-Macquart**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -26,7 +26,17 @@ function App() {
   useEffect(() => {
     api.defaults.headers.authorization = `Bearer ${cookies.get('token')}`;
     axios
-      .post(`${process.env.REACT_APP_API_URL}/auth/login/token`, {})
+
```
Comment (HeloiseFerla) [inline src/App.jsx]: empty line

File: src/App.jsx [hunk 2]
```diff
@@ -26,7 +26,17 @@ function App() {
...
+      .post(
+        `${process.env.REACT_APP_API_URL}/auth/login/token`,
+        {},
+        {
+          headers: {
+            Authorization: `Bearer ${cookies.get('token')}`,
+          },
+        }
+      )
+
```
Comment (HeloiseFerla) [inline src/App.jsx]: empty line

File: src/components/Counter/index.jsx [hunk 3]
```diff
@@ -1,21 +1,36 @@
...
   const [count, setCount] = useState(0);
   const prev = () => (count <= 0 ? 0 : setCount((prevCount) => prevCount - 1));
   const next = () => setCount((nextCount) => nextCount + 1);
 
+  useEffect(() => {
+    const newArray = cart;
+    newArray[id - 1] = count;
+    setCart(newArray);
+  }, [count]);
+
```
Comment (HeloiseFerla) [inline src/components/Counter/index.jsx]: empty line

File: src/components/Counter/index.jsx [hunk 4]
```diff
@@ -1,21 +1,36 @@
...
   return (
     <SCounter>
       <button type="button" onClick={prev}>
         -
       </button>
 
       <h3>{count}</h3>
+
```
Comment (HeloiseFerla) [inline src/components/Counter/index.jsx]: empty lines

**Review by HeloiseFerla — CHANGES_REQUESTED** [review]

**Review by HeloiseFerla — DISMISSED** [review]

File: src/pages/Supplies/index.jsx [hunk 5]
```diff
@@ -4,40 +4,75 @@ import fournitures from 'assets/Img/fournitures.png';
...
   const [supplies, setSupplies] = useState([]);
+  const [cart, setCart] = useState([]);
+  const [modal, setModal] = useState(false);
+  const fillCart = () => {
+    const newArray = [];
+    for (let i = 0; i < supplies.length; i += 1) {
+      newArray.push(0);
+    }
+    setCart(newArray);
+    console.log(cart);
```
Comment (Hacene010) [inline src/pages/Supplies/index.jsx]: ??

**Review by Hacene010 — CHANGES_REQUESTED** [review]

**Review by HeloiseFerla — DISMISSED** [review]

File: src/pages/Supplies/index.jsx [hunk 6]
```diff
@@ -38,7 +37,6 @@ export default function Supplies() {
   return (
     <>
       <SSupplies>
-        {console.log(modal)}
         <Header logo={fournitures} />{' '}
```
Comment (Hacene010) [inline src/pages/Supplies/index.jsx]: forgot this {' '}

**Review by Hacene010 — CHANGES_REQUESTED** [review]

**Review by HeloiseFerla — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**